time/src/main/java/io/spine/time/LocalDates.java

```diff
@@ -129,15 +131,17 @@ static void checkDate(LocalDate date) {
      *     <li>the month is not in the range of {@code JANUARY} to {@code DECEMBER},
      *     <li>the day is less or equal zero or greater than can be in the month.
      * </ul>
-     * @throws IllegalArgumentException if one of the arguments is invalid
+     *
+     * @throws IllegalArgumentException
+     *         if one of the arguments is invalid
      */
     private static void checkDate(int year, Month month, int day) {
```

Comment:
Is this method still used? If so, please substitute with `VBuilder`/`Validate` calls.

Reply:
It is and it cannot be substituted with an appropriate `VBuilder` or `Validate` logic, because it is not possible to validate if the day number is valid against specific Month using `VBuilder`s.